Utest/rimea_tests/reportscript.py

```diff
@@ -1,30 +1,28 @@
 import re
 import subprocess
-from pylatex import Document, Command, PageStyle, Head, MiniPage, LargeText, LineBreak, MediumText, LongTabu, NewPage, Package, Section, Description, Figure
+from pylatex import UnsafeCommand, Document, Command, PageStyle, Head, MiniPage, LargeText, LineBreak, MediumText, LongTabu, NewPage, Package, Section, Description, Figure
 from pylatex.utils import NoEscape, bold
 import datetime
 import os
 
 
 def get_evac_time(testnumber):
-	"""
-	Fetch the evacuation time from log.txt of tests
-	:param testnumber: the number of test(int)
-	:return: All evacuation times(list)
-	"""
-	log_name = "./test_" + str(testnumber) + "/log_test_" + str(testnumber) + ".txt"
+    """
```

Comment:
What's the difference?

Reply:
Didn't add the last changes to my repository, so in that file the comments are still there. 
Sorry I will take care of this next time.